data/data/bootstrap/baremetal/files/usr/local/bin/startironic.sh.template

```diff
@@ -152,11 +150,31 @@ $IPTABLES -t raw -A DHCP_IRONIC -j DROP
 {{end}}
 
 # Wait for images to be downloaded/ready
-podman wait -i 1000ms ipa-downloader
+#podman wait -i 1000ms ipa-downloader
 podman wait -i 1000ms coreos-downloader
-while ! curl --fail http://localhost/images/rhcos-ootpa-latest.qcow2.md5sum ; do sleep 1; done
-while ! curl --fail --head http://localhost/images/ironic-python-agent.initramfs ; do sleep 1; done
-while ! curl --fail --head http://localhost/images/ironic-python-agent.kernel ; do sleep 1; done
+
+# Write master network config to /tmp/nmstate for image_customization_controller to serve our merged images
+mkdir -p /tmp/nmstate
+{{range .PlatformData.BareMetal.Hosts}}
+    cat >/tmp/nmstate/{{.Name}}.yaml <<EOF
+{{.NetworkConfig}}
+EOF
+{{end}}
+
+CUSTOMIZATION_IMAGE="virthost.ostest.test.metalkube.org:5000/localimages/image-customization-controller"
```

Comment:
This needs to be referenced via image_for like IPA_DOWNLOADER_IMAGE etc at the start of the script

Reply:
hrrm, is this part of the release?  I've been building it by hand and sticking it in the local repo to pull here.. :/

Reply:
It's not part of the release yet, but it should be in the next couple of days I hope.

Reply:
Added a placeholder for now.  I'm still testing the patch after changes.

Reply:
Done now.